docs/mint.json

```diff
@@ -1,626 +1,899 @@
-          ]
-        },
-        {
-          "group": "Frontend",
-          "pages": [
-            "contributing/frontend/overview"
-          ]
-        },
-        "contributing/statistics/overview",
-        "contributing/code-of-conduct"
-      ]
-    },
-    {
-      "group": "API",
-      "pages": [
-        "api-reference/overview",
-        "api-reference/resources",
-        "api-reference/policies",
-        "api-reference/presenters"
-      ]
-    },
-    {
-      "group": "Backfills",
-      "pages": [
-        "api-reference/backfills/overview",
-        "api-reference/backfills/read-backfills",
-        "api-reference/backfills/create-backfills",
-        "api-reference/backfills/update-backfill",
-        "api-reference/backfills/delete-backfill"
-      ]
-    },
-    {
-      "group": "Blocks",
-      "pages": [
-        "api-reference/blocks/overview",
-        "api-reference/blocks/read-block",
-        "api-reference/blocks/create-block",
-        "api-reference/blocks/update-block",
-        "api-reference/blocks/delete-block"
-      ]
-    },
-    {
-      "group": "Logs",
-      "pages": [
-        "api-reference/logs/overview",
-        "api-reference/logs/read-logs"
-      ]
-    },
-    {
-      "group": "OAuth access tokens",
-      "pages": [
-        "api-reference/oauth-access-tokens/overview",
-        "api-reference/oauth-access-tokens/read-oauth-access-tokens"
-      ]
-    },
-    {
-      "group": "Pipeline runs",
-      "pages": [
-        "api-reference/pipeline-runs/overview",
-        "api-reference/pipeline-runs/read-pipeline-runs",
-        "api-reference/pipeline-runs/trigger-pipeline"
-      ]
-    },
-    {
-      "group": "Pipeline schedules",
-      "pages": [
-        "api-reference/pipeline-schedules/overview",
-        "api-reference/pipeline-schedules/read-pipeline-schedules"
-      ]
-    },
-    {
-      "group": "Pipelines",
-      "pages": [
-        "api-reference/pipelines/overview",
-        "api-reference/pipelines/read-pipeline",
-        "api-reference/pipelines/create-pipeline",
-        "api-reference/pipelines/update-pipeline",
-        "api-reference/pipelines/delete-pipeline"
-      ]
-    },
-    {
-      "group": "Sessions",
-      "pages": [
-        "api-reference/sessions/overview",
-        "api-reference/sessions/create-session"
-      ]
-    },
-    {
-      "group": "About",
-      "pages": [
-        "about/features",
-        "about/releases",
-        "about/roadmap"
-      ]
-    }
-  ],
-  "backgroundImage": "/background.png",
-  "footerSocials": {
-    "linkedin": "https://www.linkedin.com/company/magetech/",
-    "twitter": "https://twitter.com/mage_ai",
-    "facebook": "https://www.facebook.com/mageai",
-    "instagram": "https://www.instagram.com/mage_ai/"
-  },
-  "analytics": {
-    "amplitude": {
-      "apiKey": "b3c3b09423fea2abaae92c08b44de1c3"
-    }
-  },
-  "api": {
-    "playground": {
-      "mode": "hide"
-    }
-  }
+	"name": "Mage",
```

Comment:
Can you use spaces instead of tabs in this file for indentation?
We usually use 2 spaces indent in json file.

Reply:
Something changed with my JSON formatter— I'll fix